inbm/dispatcher-agent/tests/unit/sota/test_os_factory.py

```diff
@@ -33,16 +33,19 @@ def test_raise_error_unsupported_OsFactory(self) -> None:
         self.assertRaises(ValueError, factory.get_os, "MacOS")
 
     def test_create_ubuntu_snapshot_checker(self) -> None:
-        assert type(SotaOsFactory(self.mock_disp_broker, None, []).get_os('Ubuntu').create_snapshotter('update', '1', False)) \
-            is DebianBasedSnapshot
+        ubuntu_os = SotaOsFactory(self.mock_disp_broker, None, []).get_os('Ubuntu')
+        snapshotter = ubuntu_os.create_snapshotter('update', '1', False, True)
```

Comment:
**Suggestion:** Use explicit keyword arguments for clarity when calling create_snapshotter. [readability]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
        snapshotter = ubuntu_os.create_snapshotter(update_type='update', version='1', reboot=False, save_snapshot=True)
```